server/datastore/mysql/hosts_test.go

```diff
@@ -4206,3 +4207,57 @@ func testShouldCleanTeamPolicies(t *testing.T, ds *Datastore) {
 		require.Equal(t, shouldCleanTeamPolicies(c.currentTeamID, c.newTeamID), c.out)
 	}
 }
+
+func testListPoliciesForHost(t *testing.T, ds *Datastore) {
```

Comment:
Thanks for adding tests!

PS: Out of curiosity, there are already some tests for `ListPoliciesForHost` (in `server/datastore/mysql`), any specific reason you are adding these?

Reply:
I swear I searched for other tests and I didn't found them 🤦 , I will remove this as it's redundant. Thanks!